### ISSUE TITLE
consecutive empty nodes being collapsed

### ISSUE THREAD

**Issue by daaku** [tj/haml.js#6]
This haml:

```
!!!5
%html
  %body
    #empty-one
    #empty-two
```

gives this html:

```
<!DOCTYPE html> 
<html> 
<body> 
<div id="empty-two"></div> 
</body> 
</html> 
```

**Comment by tj** [comment]
should be a simple fix I will look into it

**Comment by tj** [comment]
ah, currently its because #one and #two are technically on the same "block" so for example if you:
    .foo
    .bar
    .baz

it is read as
    <div class="foo bar baz">

since the engine does not differentiate between newlines and indents ATM.
Right now you can use this instead until a fix is finished:
    %div#empty-one
    %div#empty-two

**Comment by tj** [comment]
Fixed empty tags followed by class / ids on new lines. Closed by 9bbc55455cbc0a9ed985e5377561141927580e48

quick hack but it works :)

**Comment by daaku** [comment]
haha, nice. thanks TJ
